### DISCUSSION TITLE
flyte binary vs core

### DISCUSSION THREAD

**Discussion by sorushsaghari** [flyteorg/flyte#5728] [Q&A] [answered]
I have tried to find the real difference between flyte-core and binary helm charts. but there is no official hints about them. i dont know which one to choose. the only thing makes me to use flyte core is that flyte-binary docker image has no realese tag conraining version like flyte-core components. could you please help me ?

**Comment by davidmirror-ops** [comment, marked as answer]
I was thinking exactly about this! (I mean, the need to clarify in the docs the differences)
Tha main motivator to choose flyte-core over flyte-binary is multi (K8s) cluster, that is only supported on flyte-core. Also with flyte-core, you have the possibility of scaling out components independently, which may be helpful in scenarios of high load.
Could you file an issue to add a better explanation to the docs?
Does this answer your question in some way?

**Reply by sorushsaghari** [reply]
yeah. my Thank you for the clarification! Your explanation about Flyte-core's advantages for multi-cluster support and independent scaling makes sense, and it does answer my initial question.

However, I still have a concern about Flyte-binary: the lack of release tags containing versions in the Flyte-binary Docker images in [flyte packages](https://github.com/flyteorg/flyte/pkgs/container/flyte-binary) makes it challenging to track or use specific versions. Could you provide more details on how to manage Flyte-binary versions, or is there a recommended approach for this?

Filing an issue to improve the documentation on these differences sounds like a good idea, and I'd be happy to do that
